### DISCUSSION TITLE
FEATURE REQUEST: sort results in search #457????

### DISCUSSION THREAD

**Discussion by awarnick1** [dailydotdev/daily#948] [Feature Requests]
Is this feature going to implemented any time soon? It's been 18 months since this was requested. Seems like a simple feature to implement. It's also annoying in the search results as you only see old news at the top and it's almost impossible to find what you want to find in the search. May as well not even have a search feature.

**Comment by idoshamun** [comment]
Hi @awarnick1,

I understand your frustration about our search 😥.
I replied on the original thread https://github.com/dailydotdev/daily/discussions/294#discussioncomment-6648773.
Let's keep the discussion there please
